plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -404,6 +405,11 @@ public IcebergSessionProperties(
                         "Enable file-based conflict detection: take partition information from the actual written files as a source for the conflict detection system",
                         icebergConfig.isFileBasedConflictDetectionEnabled(),
                         false))
+                .add(integerProperty(
+                        MAX_PARTITIONS_PER_WRITER,
+                        "Max count of partitions per writer",
```

Comment:
Let's align this with the `IcebergConfig` property description and use: `"Maximum number of partitions per writer"`